packages/jest-diff/src/__tests__/diff.test.js

```diff
@@ -164,3 +164,29 @@ test('collapses big diffs to patch format', () => {
 
   expect(result).toMatchSnapshot();
 });
+
+describe('context', () => {
+  const _test = (contextLines?: number) => {
+    test(`number of lines: ${typeof contextLines === 'number'
+      ? contextLines
+      : 'null'} ${typeof contextLines !== 'number' || contextLines < 0
+      ? '(5 default)'
+      : ''}`, () => {
```

Comment:
Flow complains if I extract it and I did not simplify it because 0 is falsy too.